symengine/ruby/lib/symengine/integer.rb

```diff
@@ -0,0 +1,93 @@
+class Fixnum
```

Comment:
I was looking for the exact same thing! Thanks! Now I think this part of code won't be needed.